pkg/process/config/yaml_config.go

```diff
@@ -50,6 +50,12 @@ func (a *AgentConfig) loadSysProbeYamlConfig(path string) error {
 		a.CollectDNSStats = config.Datadog.GetBool(key(spNS, "collect_dns_stats"))
 	}
 
+	if config.Datadog.IsSet(key(spNS, "collect_dns_domains")) {
+		fmt.Println("Setting collect dns domains")
```

Comment:
Are the `Println`'s left over from debugging?

Reply:
Yes 🤦 . Will remove. Thanks!